src/App.js

```diff
@@ -1,14 +1,47 @@
-import React, { useState } from 'react';
+import React, {useState} from 'react';
+import {
+    Switch,
+    Route,
+    Link
+} from 'react-router-dom';
+
 import './App.css';
+import Home
+    from "./components/Home";
+import Login
+    from "./components/Login";
+import Blogposts
+    from "./components/Blogposts";
 
 function App() {
-  // We houden in de state bij of iemand is "ingelogd" (simpele versie)
-  const [isAuthenticated, toggleIsAuthenticated ] = useState(false);
+    const [isAuthenticated, toggleIsAuthenticated ] = useState(false);
 
   return (
-    <div>
-      Maak hier jouw prachtige blog-applicatie!
-    </div>
+      <>
+          <nav className="nav-bar">
+              <ul>
```

Comment:
zag in 1 van je commit de vraag waar de margin vandaan komt. Dat komt door de `ul` hie zit standaard margin op.